m4_sessions/1705-inning/group_four.md

```diff
@@ -1,10 +1,31 @@
-# Group Four
+# Lending Privilege
+## Plan
 
-## List your group members here:
-- Katy Farmer
-- Edilene Da Cruz
-- Staci McWilliams
+* 20 - Videos
+* 15 - Guest Speaker(pending) || Individual Reading and Reflection
+* 20 - Small Group Discussion
+* 5 - Share Out 
+
+## Videos
+* [Intro video](https://www.youtube.com/watch?v=SqL9onVybW0) 2:53
+* [The Power of Privledge](https://www.youtube.com/watch?v=N0acvkHIiZs) 14:57
+
+## Guest Speaker || Reading
+* Pending Guest Speaker (estimated time 20 minutes) 20:00
+* [For white people who believe colorblindness is enough](http://www.huffingtonpost.com/entry/for-white-people-who-believe-colorblindness-is-enough_us_5782a4b0e4b0f06648f526a1?section=) 10:00 and add time to group discussion
+
+## Outcome 
+* Awareness of privlilege and how to be an ally to peers.
 
-- Sesson Title - Lending Privilege.
-- Awareness of privlilege and how to be an ally to peers.
-- Two ten minute videos, 10 minute individual reflection and then 20 minute small group discussions, guest speaker(10 minutes), 10 minute wrap up reflection. 
+## Questions and Individual Reflection 10:00
+* What's true about these videos?
+* What has been your personal experience with privledge?
```

Comment:
Maybe use some of the language from the TED talk..."What are you missing that is right in front of you?"  "Is privilege something you have or don't have?  When are you turing on the switch?"